src/utils/checkError.js

```diff
@@ -0,0 +1,42 @@
+import ErrorMessage from '../constants/ErrorMessage.js';
+import { CarInfo } from '../constants/racingConfig.js';
+
+// carNameInput 관련 에러체크
+export const isNoNameInput = (nameInput) => {
+  if (nameInput == '' || nameInput === undefined || nameInput === null) {
+    throwError(ErrorMessage.PREFIX, ErrorMessage.NO_NAME_INPUT);
```

Comment:
에러 메시지를 상수화 하여 관리하는데 에러 메시지 안에 `ErrorMessage.PREFIX`를 사용하지 않으신 이유가 궁금합니다! 
코드를 보았을 때, `ErrorMessage.NO_NAME_INPUT`안에 `PREFIX`가 들어있으면 해당 코드가 시지를 상수화 하여 관리하는데 에러 메시지 안에 `ErrorMessage.PREFIX`를 사용하지 않으신 이유가 궁금합니다! 아래 처럼, 조금 더 명확하게 어떤 에러를 나타내는지 확인할 수 있다고 생각합니다! 
```suggestion
    throwError(ErrorMessage.NO_NAME_INPUT);
```

Reply:
과제 요구 사항에 "[ERROR]"로 시작하는 에러를 내뿜는다라고 되어 있기에 해당 부분을 `PREFIX`로 저장해 관리할 필요성을 느꼈습니다. 이를 에러 메세지에 무조건 포함시키게 하기보다는 에러 메세지와 과제로 지정된 접두어를 따로 분리하는 게 더 좋아보여서 `MESSAGE`와 `PREFIX`를 분리하게 됐습니다.

다만, 에러메세지에 PREFIX를 포함시키지 않더라도 명확성을 위해 `throwError()`를 살짝 수정시키는 게 큰 개선방법이 될 수 있어보입니다!

```js
throwError(ErrorMessage.NO_NAME_INPUT);

// 에러 처리 함수
const throwError = (message) => {
  throw new Error(`${ErrorMessage.PREFIX} ${message}`);
};
```
이렇게 하면 메세지와 PREFIX를 분리하면서도, 명확성을 챙길 수 있도록 수정할 수 있겠네요🙂
좋은 피드백 감사합니다.